pandas/tests/series/test_ufunc.py

```diff
@@ -453,3 +453,18 @@ def test_np_matmul():
         expected_result,
         result,
     )
+
+
+def test_array_ufuncs_for_many_arguments():
+    # GH39853
+
+    def add3(x, y, z):
+        return x + y + z
+
+    ufunc = np.frompyfunc(add3, 3, 1)
+    s = pd.Series(np.array([1, 2]))
```

Comment:
We try to avoid using single letter variables, ser would be preferred here imo.

Reply:
Ok. Noted.